app/models.py

```diff
@@ -1448,6 +1485,8 @@ def status(self):
     def status(cls):
         return sql_case([
             (cls.submitted_at.isnot(None), 'submitted'),
+            (cls.awarded_at.isnot(None), 'awarded'),
```

Comment:
The test for this seems to pass OK? `test_awarded_status_for_brief_response_has_been_awarded`